frontend/schema.graphql

```diff
@@ -31,18 +31,18 @@ type AcceptingStore {
   categoryId: Int!
   contact: Contact!
   contactId: Int!
-  description: String!
+  description: String
   id: Int!
-  name: String!
+  name: String
   physicalStore: PhysicalStore
 }
 
 type Address {
   id: Int!
-  location: String!
-  postalCode: String!
-  state: String!
-  street: String!
+  location: String
+  postalCode: String
+  state: String
+  street: String
```

Comment:
### Introducing nullables to a not null-safe frontend
![I dont care](https://media.giphy.com/media/QMHoU66sBXqqLqYvGO/giphy.gif)

(just kidding, I'm fine with it^^)

Reply:
😄 Null safety is in beta since 10/2020, I hope they will finally push it to stable in the next weeks …